content/rc/subscriptions/maintenance/_index.md

```diff
@@ -37,3 +36,6 @@ Urgent maintenance refers to any activity that could affect service and cannot w
 
 Redis can perform urgent maintenance at any time, even if you have set a manual maintenance window or have temporarily skipped maintenance. Redis will notify users by email when urgent maintenance starts and ends.
 
+### Major upgrades
+
+For major upgrades or upgrades that might include breaking changes, users will receive an advance notification before the upgrade.
```

Comment:
@rrelledge, I would add that the advanced notification will be received with sufficient time for the customers to get prepared to the upgrade. Something like that. It's just I'm predicting that customers will ask "how much in advance"

Reply:
Yes, that does sound likely. I updated this section to include this.